src/commands/build.rs

```diff
@@ -120,133 +121,95 @@ impl BlueBuildCommand for BuildCommand {
-
-        let container_file_path = Path::new(CONTAINER_FILE);
-
-        if !self.force && container_file_path.exists() {
-            let gitignore = fs::read_to_string(GITIGNORE_PATH)
-                .context(format!("Failed to read {GITIGNORE_PATH}"))?;
-
-            let is_ignored = gitignore
-                .lines()
-                .any(|line: &str| line.contains(CONTAINER_FILE));
-
-            if !is_ignored {
-                let containerfile = fs::read_to_string(container_file_path)
-                    .context(format!("Failed to read {}", container_file_path.display()))?;
-                let has_label = containerfile.lines().any(|line| {
-                    let label = format!("LABEL {BUILD_ID_LABEL}");
-                    line.to_string().trim().starts_with(&label)
-                });
-
-                let question = requestty::Question::confirm("build")
-                    .message(
-                        if has_label {
-                            LABELED_ERROR_MESSAGE
-                        } else {
-                            NO_LABEL_ERROR_MESSAGE
-                        }
-                        .bright_yellow()
-                        .to_string(),
-                    )
-                    .default(true)
-                    .build();
-
-                if let Ok(answer) = requestty::prompt_one(question) {
-                    if answer.as_bool().unwrap_or(false) {
-                        blue_build_utils::append_to_file(
-                            &GITIGNORE_PATH,
-                            &format!("/{CONTAINER_FILE}"),
-                        )?;
-                    }
-                }
-            }
-        }
+        self.update_gitignore()?;
 
         if self.push && self.archive.is_some() {
             bail!("You cannot use '--archive' and '--push' at the same time");
         }
 
-        let recipe_path = self.recipe.clone().unwrap_or_else(|| {
+        if self.push {
+            blue_build_utils::check_command_exists("cosign")?;
+            check_cosign_files()?;
+        }
+
+        let recipe_paths = self.recipe.clone().map_or_else(|| {
             let legacy_path = Path::new(CONFIG_PATH);
             let recipe_path = Path::new(RECIPE_PATH);
             if recipe_path.exists() && recipe_path.is_dir() {
-                recipe_path.join(RECIPE_FILE)
+                vec![(CONTAINER_FILE.to_string(), recipe_path.join(RECIPE_FILE))]
             } else {
                 warn!("Use of {CONFIG_PATH} for recipes is deprecated, please move your recipe files into {RECIPE_PATH}");
-                legacy_path.join(RECIPE_FILE)
+                vec![(CONTAINER_FILE.to_string(), legacy_path.join(RECIPE_FILE))]
             }
+        },
+        |recipes| {
+            recipes.into_par_iter().filter_map(|recipe| {
+                Some((format!("{CONTAINER_FILE}.{}", recipe.file_stem()?.to_str()?), recipe))
```

Comment:
Maybe use the recipe path instead and replace `/` with `_`

Reply:
Went with a hash of the recipe file path instead so each Containerfile will have only one recipe.